src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -53,7 +53,8 @@ function LtiConfigForm({ onSubmit, intl, formRef }) {
   const isInvalidConsumerSecret = Boolean(touched.consumerSecret && errors.consumerSecret);
   const isInvalidLaunchUrl = Boolean(touched.launchUrl && errors.launchUrl);
   const supportEmail = getConfig().SUPPORT_EMAIL;
-  const showLTIConfig = user.administrator || !app.adminOnlyConfig;
+  const showLTIConfig = user.administrator;
```

Comment:
lti config will be visible only on admin check but above in the code snippet, 
 `const canSubmit = getAuthenticatedUser().administrator || !app.adminOnlyConfig`
 the user can still submit if it is not admin only config. bit confused about the logic

Reply:
`const showLTIConfig = user.administrator || !app.adminOnlyConfig;` this code snippet is removed. Now updated condition is `const showLTIConfig = user.administrator;`